server/service/src/sync/api/common_records.rs

```diff
@@ -9,24 +9,22 @@ fn empty_object() -> serde_json::Value {
 }
 
 #[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
-pub(crate) struct CommonSyncRecordV5 {
-    #[serde(rename = "tableName")]
+#[serde(rename_all = "camelCase")]
```

Comment:
I was changing SyncAction from SyncActionV5, and decided to just use rename_all, another sidetrack sorry